tests/e2e/pageobjects/dashboard/Dashboard.ts

```diff
@@ -63,11 +63,29 @@ export class Dashboard {
     }
 
     async waitPage(timeout: number = TestConstants.TS_SELENIUM_LOAD_PAGE_TIMEOUT) {
+        const polling: number = TestConstants.TS_SELENIUM_DEFAULT_POLLING;
+        const attempts: number = Math.ceil(timeout / polling);
         Logger.debug('Dashboard.waitPage');
 
-        await this.driverHelper.waitVisibility(By.css(Dashboard.WORKSPACES_BUTTON_CSS), timeout);
-        await this.driverHelper.waitVisibility(By.css(Dashboard.STACKS_BUTTON_CSS), timeout);
-        await this.driverHelper.waitVisibility(By.xpath(Dashboard.GET_STARTED_BUTTON_XPATH), timeout);
+        let workspacesLocated: boolean = false;
```

Comment:
I am not sure that this changes fix a root cause of problem.
Here we can see that timeout provided to method ("waitVisibility") is "TS_SELENIUM_LOAD_PAGE_TIMEOUT" (180 seconds by default) and in this context, it is not understandable why method failed with timeout 1s (as you noted in the PR description). Maybe the root problem inside of the "waitVisibility" method?

Reply:
The method threw TimeoutError because it could not find the element it was looking for

Reply:
and since nothing was catching for exceptions there, the TimeoutException from waitVisibility caused the entire method to fail